packages/common/src/core/schemas/internal/getEditorSchemas.ts

```diff
@@ -353,12 +353,16 @@ export async function getEditorSchemas(
           import("../../../events/_internal/EventSchemaCreate"),
         "hub:event:edit": () =>
           import("../../../events/_internal/EventSchemaEdit"),
+        "hub:event:attendees": () =>
+          import("../../../events/_internal/EventSchemaEdit"),
```

Comment:
Per my other note, since those fields aren't needed for `Details` pane, may be good to split them into their own separate schema.

```ts
"hub:event:attendees": () =>
          import("../../../events/_internal/EventSchemaAttendees"),
```